modules/ROOT/pages/configure-uri-template-regex.adoc

```diff
@@ -0,0 +1,81 @@
+= Configure URI Template Regex
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: policy, custom, ootb, offline, resource level
+
+//When you do {blah blah}, you must configure the URI template regex. -why do we need to confiugure regex?
+
+When configuring the URI template regex, the base path of the Mule runtime engine (Mule) application that implements (or proxies) the API must be considered. This is because Mule 4 resource-level policies are dependent on the location where an API is deployed.
+
+Let’s look at the following example to understand this better:
+
+image:dhsdfksh.png[]
+
+Imagine that this API specification is implemented in the following Mule application:
++
+[source,text,linenums]
+----
+
+<api-gateway:autodiscovery apiId="1" flowRef="proxy"/>
+
+<http:listener-config name="httpConfig" basePath="api">
+   <http:listener-connection host="localhost" port="80"/>
+</http:listener-config>
+
+<flow name="proxy">
+   <http:listener config-ref="httpConfig" path="v1/*" />
+   <logger />
+</flow>
+----
+
+In this example, the `HTTP listener-config` element defines a basePath attribute and the HTTP listener element also defines an additional base path, v1, before the trailing ”*” in the path. The concatenation of these two paths is referred to as base path.. The base path in the example now is `/api/v1`.
```

Comment:
Extra "." after "referred to as base path".

Reply:
Done! Thanks!